### PR TITLE
Update the documentation to include the new emissions endpoints.

### PR DESCRIPTION
**Pull request by alawTrackUnit**

### CODE REVIEW

File: api-docs/emissions-api.md [hunk 1]
```diff
@@ -3,18 +3,26 @@ title: Emissions API - Introduction
...
-#### There are 3 endpoints currently available:
+#### There are 5 categories currently available:
 1. Monthly: Aggregated 12 values per asset, 1 for each of the last 12 months
 2. Lifetime: Aggregated 1 value per asset representing the total accumulation since the machine was originally put in use. This value is updated once a day
 3. Daily: This is a non-aggregated limited set of most recent values (ex. 60 days). It provides one value per day for each of the recent days.
+4. Period: This is an aggregated set of values for a specific period of time. The current periods can be returned by previous day, previous seven days, or previous thirty days. It provides one value per period for each of the days in the period.
+5. Sites: This is an aggregated set of values for a specific site. It provides one value per month for as long as the site has existed. Site Ids are returned by the Sites endpoint.
+
+#### KPI Endpoints
+* Returns Emissions, Total Fuel, Total Emissions, Total Usage Hours, Total kWh, Fuel Compilation, Emissions Compilation, Usage Hours Compilation, kWh Compilation, Average Emissions, Average Emissions by Usage Hours, Different Power Types, grouped by asset types
```
Comment (haoyukongTrackunit) [inline api-docs/emissions-api.md:22]: Kwh insteads of kwh (Upper case K to make it consistent)

File: changelogs/emissions-api-update-sites-filtering-12-12-2023.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Added new Endpoints to the Emissions API to enable filtering by assets, as well as getting emissions value for sites.
+
+Below is a list of all the new endpoints:
+
+> ➕ GET: /emissions/period
+
+[Get period](ref:getemissions)
+
+> ➕ POST:
```
Comment (haoyukongTrackunit) [inline changelogs/emissions-api-update-sites-filtering-12-12-2023.md]: should we put a URL (such as: emissions/period) after each GET and POST?

File: api-docs/emissions-api.md [hunk 3]
```diff
@@ -3,18 +3,26 @@ title: Emissions API - Introduction
...
 >
 > This API is available for customers that have acquired a license for the Emissions Reporting capability.
 
 ## Interface
 
-#### There are 3 endpoints currently available:
+#### There are 5 categories currently available:
 1. Monthly: Aggregated 12 values per asset, 1 for each of the last 12 months
 2. Lifetime: Aggregated 1 value per asset representing the total accumulation since the machine was originally put in use. This value is updated once a day
 3. Daily: This is a non-aggregated limited set of most recent values (ex. 60 days). It provides one value per day for each of the recent days.
```
[the remark this answers is not in the document]
Comment (alawTrackUnit) [reply api-docs/emissions-api.md:17]: This is explained on the actual daily pages. This is an example in the intro.

File: changelogs/emissions-api-update-sites-filtering-12-12-2023.md (hunk 2 above)
Comment (alawTrackUnit) [reply changelogs/emissions-api-update-sites-filtering-12-12-2023.md]: This has been added.

**Review by haoyukongTrackunit — APPROVED** [review]

**Review by zkh1212 — APPROVED** [review]
Looks good.

**Review by KateVomBerg — APPROVED** [review]
Great work, feel free to merge 👍

**Pull request merged**